src/app/lib/app_router.dart

```diff
@@ -9,7 +9,7 @@ import 'package:get_it/get_it.dart';
 import 'package:go_router/go_router.dart';
 import 'package:logger/logger.dart';
 
-final _rootNavigatorKey = GlobalKey<NavigatorState>();
+final rootNavigatorKey = GlobalKey<NavigatorState>();
```

Comment:
make it private may be?

Reply:
it's needed outside of this file. Especially so alice inspector works